dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/DateFormatProvider.java

```diff
@@ -14,11 +14,13 @@
 import java.text.SimpleDateFormat;
 
 public class DateFormatProvider {
+    /** Simple date format for file names: use only while synchronized */
+    public final static SimpleDateFormat FILE_DATE = new SimpleDateFormat("_yy.MM.dd_HH.mm.ss");
```

Comment:
Since it is only used by FileLogSet, should it be made default protected instead of public?

Reply:
Yes good catch.